### PR TITLE
Aw admin order #114

### LINKED ISSUE #114: Story 19: Admin Views an Order
As an authenticated Admin, when I visit an individual order page
      Then I can see the order's date and time.
      And I can see the purchaser's full name and address.
      And I can see, for each item on the order:
          - The item's name, which is linked to the item page.
          - Quantity in this order.
          - Price
          - Line item subtotal.
      And I can see the total for the order.
      And I can see the status for the order.

### PR DESCRIPTION
**Pull request by wlffann**
Closes #114 

Needs all reviewers @kheppenstall @tmikeschu @akintner 

This allows an admin to see an order's information, including user name and email address. It also authenticates a regular user when they attempt to view an order.

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -13,4 +14,5 @@ def create
     flash[:success] = "Order was successfully placed"
     redirect_to orders_path
   end
+
```
Comment (akintner) [inline app/controllers/orders_controller.rb]: can you jump in and delete this extra line really quickly?

**Comment by akintner** [comment]
One little thing, Hound-related: want to jump into those Rspec tests and change all the double quotes to singles? Super tiny, I realize, but super fast to change. 

Otherwise, this looks great!

**Comment by wlffann** [comment]
Oy, I totally missed that. I'll change them now, thanks!

File: spec/features/admin/visit_another_users_order_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    expect(page).to have_content(order.status)
+    order.items.each do |item|
+      expect(page).to have_content(item.name)
+      expect(page).to have_content(order.order_item_quantity(item))
+      expect(page).to have_content(order.order_item(item).current_price_per_unit)
+      expect(page).to have_content(order.subtotal(item))
+    end
+  end
+
+  scenario 'visitor cannot see order details' do
```
Comment (tmikeschu) [inline spec/features/admin/visit_another_users_order_spec.rb]: Can we move the user and visitor context to different specs?

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -3,6 +3,7 @@ class SessionsController < ApplicationController
   skip_before_action :require_login, only: [:new, :create]
 
   def new
+    redirect_to dashboard_path if current_user
```
Comment (wlffann) [inline app/controllers/sessions_controller.rb]: Thanks!

**Comment by kheppenstall** [comment]
👍 Looks all good to me!

File: app/models/order.rb [hunk 4]
```diff
@@ -31,4 +31,8 @@ def checkout(cart)
     end
   end
 
+  def verified_user?(current_user)
```
Comment (wlffann) [inline app/models/order.rb]: Great suggestion, thanks!

File: spec/features/admin/visit_another_users_order_spec.rb [hunk 5]
```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.feature 'Visit another users order' do
+
+  scenario 'admins can see all order details' do
+    admin = create(:user, role: 1)
+    user = create(:user_with_orders)
```
Comment (wlffann) [inline spec/features/admin/visit_another_users_order_spec.rb]: Got it!

**Pull request merged**